### PR TITLE
Css redesign

### PR DESCRIPTION
**Pull request by DrAnax**

### CODE REVIEW

File: client/src/lobby/keyboardInit.ts [hunk 1]
```diff
@@ -30,8 +30,8 @@ export default function keyboardInit(): void {
       if (globals.currentScreen === Screen.Lobby) {
         $(".lobby-games-join-first-table-button").click();
       }
-    } else if (event.altKey && (event.key === "c" || event.key === "ç")) {
-      // Alt + c
+    } else if (event.altKey && (event.key === "n" || event.key === "ç")) {
```
Comment (padiwik) [inline client/src/lobby/keyboardInit.ts:33]: and the c cedille?

File: client/src/lobby/nav.ts [hunk 2]
```diff
@@ -57,6 +57,43 @@ export function init(): void {
...
   });
 
+  // Games bottom screen toggle button
+  $("#lobby-toggle-show-tables").on("click", () => {
+    $("#lobby-toggle-show-tables").addClass("toggle-active");
+    $("#lobby-toggle-show-chat").removeClass("toggle-active");
+    $("#lobby-toggle-show-game-chat").removeClass("toggle-active");
+    $("#lobby-top-half").addClass("toggle-active");
+    $("#lobby-bottom-half").removeClass("toggle-active");
+    // restore online users and chat for small screens
```
Comment (Zamiell) [inline client/src/lobby/nav.ts:67]: capitalize comments, match the existing comment style

File: client/src/lobby/keyboardInit.ts (hunk 1 above)
Comment (padiwik) [reply client/src/lobby/keyboardInit.ts:33]: ñ

Comment (DrAnax) [reply client/src/lobby/keyboardInit.ts:33]: Not sure what the second key is for; my guess was that it is mac specific. That's why I need a fix on line 33.

File: public/css/hanabi.css [hunk 3]
```diff
@@ -833,72 +836,75 @@ input::-webkit-calendar-picker-indicator {
 }
 
 /*
-  Media edits
+  Media Responsive Utilities
```
Comment (Zamiell) [inline public/css/hanabi.css:839]: good, thank you for documenting this

File: client/src/lobby/keyboardInit.ts (hunk 1 above)
Comment (padiwik) [reply client/src/lobby/keyboardInit.ts:33]: Yeah it's mac specific https://www.webnots.com/option-or-alt-key-shortcuts-to-insert-symbols-in-mac-os-x/ so replace the ç with ˜ i guess

**Comment by DrAnax** [comment]
done

**Comment by Zamiell** [comment]
the breakpoint at 1280px seems off, can you confirm?
like, the new game button isn't even close to the "Hanab Live"

same thing with the breakpoint at 980px. it isn't as bad as the 1280px one, but it still needs to be adjusted

same with the one at 840px

**Comment by Zamiell** [comment]
basically it should only break when the buttons start to overlap, or are at least close to overlapping

**Comment by DrAnax** [comment]
Check again please

**Comment by Zamiell** [comment]
better

but now the "Hanab Live" text disappears at 1065px

i think it should only disappear at something like 920px, e.g. when it gets close to overlapping

**Comment by DrAnax** [comment]
Silly travis, I only touched a css file...

**Comment by DrAnax** [comment]
All my cats agree, it should be purfect now

**Comment by Zamiell** [comment]
yup, looking better

is it a bug that the rounded corners are removed in the mobile box?

**Comment by DrAnax** [comment]
rounded corners of which element?

**Comment by DrAnax** [comment]
It's an alpha feature. I guess it's meant to save screen space.

**Comment by DrAnax** [comment]
alpha.css:3830

**Comment by Zamiell** [comment]
i think you are wrong

just look at https://html5up.net/alpha

click on "elements", and then drag the window smaller, and you will see that even on mobile, it is supposed to look rounded

**Comment by Zamiell** [comment]
anyways, the non-rounded corners look buggy, i think that the corners should always be rounded so that it looks consistent, if that is possible to do

**Comment by DrAnax** [comment]
That's not what we use for background. We use the `box` class, an example of which can be found at https://html5up.net/alpha (it's the featured image container)

**Comment by DrAnax** [comment]
Yes, it is

**Comment by Zamiell** [comment]
this must have taken a long time to do, but its a huge upgrade for the website. very nice work!

**Pull request merged**